json_logging/framework/fastapi/__init__.py

```diff
@@ -0,0 +1,137 @@
+# coding=utf-8
+import logging
+
+from starlette.middleware.base import BaseHTTPMiddleware, RequestResponseEndpoint
+from starlette.requests import Request
+from starlette.responses import Response
+from starlette.types import ASGIApp
+
+import json_logging
+import json_logging.framework
+from json_logging.framework_base import AppRequestInstrumentationConfigurator, RequestAdapter, ResponseAdapter
+
+from json_logging.util import is_not_match_any_pattern
+
+
+def is_fastapi_present():
+    # noinspection PyPep8,PyBroadException
+    try:
+        import fastapi
+        import starlette
+        return True
+    except:
+        return False
+
+
+if is_fastapi_present():
```

Comment:
this method is declared twice

Reply:
No, the second one is an if statement using the function. This patten I adopted from other backends. 
But your comment reminds me of this mechanism and that I did not stick to the lazy importing it was for. I have to adjust the way I do the imports. Otherwise json-logging can only be used if fastapi is installed.
Will take me the one or other day until I find the time. 
So thanks for pointing me at it!

Reply:
its total fine, somehow i misunderstood it LOL